views/js/previewer/plugins/button.tpl

```diff
@@ -0,0 +1,19 @@
+<li
```

Comment:
For screen readers it is better to avoid putting buttons in lists if there are 1-3 buttons. Less verbal. For long list of buttons it is better to go with `ul`, as it is now. Is it possible for previewer have many buttons in one list?